tex/chapters/07-ApplyingModel.tex

```diff
@@ -0,0 +1,197 @@
+%!TEX root = ../main.tex
+
+\chapter{Application of the Model}\label{cha:applying-model}
+
+In this Chapter a method for assessing the sensitivity of the model will be outlined.
+This is done by making several small changes to the parameters of strategies and observing whether the model can discriminate between the new altered vectors.
+The size of the change made to the strategy will be varied and the models performance recorded for each deviation.
+It is expected that there is a cut-off where the model changes from performing well to performing badly, and this gives an approximation of it's sensitivity.
+Finally, model will be applied to the Axelrod-Python library to determine if it contains any duplicate strategies.
+
+
+
+\section{MemoryOne Strategies}
+
+A MemoryOne strategy bases its decision entirely on the the last round of play.
+In Axelrod-Python they are defined by a vector $v$ of length 4 that determines the probability of Cooperation given the previous rounds history, ie:
```

Comment:
Nope, they're defined that way throughout the literature. You'll be able to find some references about them (Stewart and Plotking 2012 I believe is a good basic one).